code/game/machinery/ARES/ARES_procs.dm

```diff
@@ -54,6 +60,14 @@ GLOBAL_LIST_INIT(maintenance_categories, list(
 		alert.delink()
 	..()
 
+/datum/ares_link/proc/change_iff(selection)
+	faction_label = selection
+	var/list/new_iff = faction_options[selection]
+
+	faction_group = new_iff
+	ares_apollo_talk("Security IFF systems updated to [selection]")
+	for(var/obj/structure/machinery/defenses/sentry/premade/deployable/almayer/mini/ares/sentry in core_sentries)
```

Comment:
as anything

Reply:
But why

Reply:
I don't see the what point 'as anything' even has